addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -530,9 +530,16 @@
                     font-weight: bold;
                     padding: 1px;
                 }
+
+                .link-custom-color-border .dropdown-menu.show {
+                    // Compensate positioning to center the border dropdown menu
+                    left: -$o-we-sidebar-content-padding-base !important;
```

Comment:
Having this rule on the dropdown will create an overflow on the `border-color` colorpicker.

I'd rather expect this rule to be applied to the others colorpicker (text-color and fill-color) as shown on the second screenshot, wdyt?

| Border colorpicker (your PR) | Fill color colorpicker (your PR) |
|--------|--------|
| ![image](https://github.com/odoo/odoo/assets/128030743/02ac8510-6b71-4832-a590-41971c175a0e) | ![image](https://github.com/odoo/odoo/assets/128030743/74d0b80a-9ea1-421b-9b60-046261ebf450) |

Reply:
Indeed, thanks. It seems the moz-available was providing weird result on firefox >125, I completely changed the way we calculte the dropdown-menu width to avoid this.